### ISSUE TITLE
Conditional ZimFile size color on Library

### ISSUE THREAD

**Issue by 4shutosh** [kiwix/kiwix-android#1989]
**Describe the feature you'd like**

Displaying  appropriate text color of ZimFile size according to 
* storage available on users device 
                      **or**
* setting a limit of size ( if file is more than 1GB or so ) will act as a warning.

**Usage**
![Untitled design](https://user-images.githubusercontent.com/53831987/78009551-d243cc00-735e-11ea-8c4c-10d21c9808ef.png)

This will not only make size more visible but also increase the attention of user on it.
As a user with limited internet access the size of download (zimfile) is a definite point to consider.
(Although there is a warning if we are using mobile data, it is only once and there is a good chance that user is still on the mobile data and does not remember about it.)

**Additional context**
I would like to work on this.

**Comment by macgills** [comment]
@kelson42 I'll leave this to your decision. I don't think we need it

**Comment by kelson42** [comment]
@macgills Online books which can not be downloaded should be fully greyed (We have or had an issued about that). So just changing the color seems inappropriate.

**Comment by macgills** [comment]
@kelson42 only items that were unsupported by the filesystem were greyed out. We only check "Insufficient space" when a user clicks on an item

**Comment by kelson42** [comment]
@macgills Then this ticket should muted to grey them out (like for the other which can not be downloaded) IMO.

**Comment by macgills** [comment]
The problem then becomes the available storage changing over time. We would need to on an interval check the available space and then make the decision to gray them out

**Comment by 4shutosh** [comment]
What should be done then? 
Should I close this Issue?

**Comment by macgills** [comment]
Depends on @kelson42 's decision. If we want to gray out files based on available storage it can be done, just needs some definition

**Comment by kelson42** [comment]
@macgills I think I'm pretty much again the idea. An appropirate library should just allow to sort by size OR/AND filter by size. We don't need this color stuff and threshold system IMO. Would be in favour of closing this as WONTFIX.

**Comment by macgills** [comment]
If an item can not be downloaded because the user's filesystem does not support files over 4GB then we mark it as can not download.
That is the only scenario where we mark files as "Can't be downloaded"
